README.md

```diff
@@ -22,6 +22,40 @@ om.insert(1,1);
 // Order of keys is [2, 1]
 ```
 
+Here, another example that shows a map of `QString` > `int`:
+
+```C++
+OrderedMap<QString, int> map;
+
+// put some values
+map["z-key"] = 10;
+map["a-key"] = 20;
+map["c-key"] = 15;
+
+// edit some values
+map["a-key"] = 21;
+
+QList<int> values = map.values();
+for (int i = 0; i < values.size(); i++) {
```

Comment:
How about we go a _little_ fancy and show the use of `foreach` with this container? ex:
```
foreach (int value, values) {
    qDebug() << value;
}
```
(`foreach` can be used because this container supports the `const_iterator`).

Reply:
OK!  :thumbsup: